server/routes/api/section.js

```diff
@@ -159,4 +165,45 @@ router.get('/updateSection', (req, res) => {
     });
 });
 
-module.exports = router;
\ No newline at end of file
+module.exports = router;
+
+// ------------- Combined Seating Layout and Course Methods -------------
+
+// gets information regarding a course given a student/prof id
+router.get('/getCourseView', (req, res) => {
+    // this is the professor ID
```

Comment:
Here's a little copy paste typo.
```suggestion
// gets information regarding a course given a course id
router.get('/getCourseView', (req, res) => {
```